src/test/java/org/folio/processing/mapping/mapper/writer/marc/MarcBibRecordModifierTest.java

```diff
@@ -291,7 +285,7 @@ public void shouldNotAdd9SubfieldToControllableField() throws IOException {
       "{\"100\":{\"subfields\":[{\"a\":\"electronic\"},{\"0\":\"test0\"},{\"9\":\"bdbf59b7-913b-42ac-b1c6-e50ae7b00e6a\"}],\"ind1\":\" \",\"ind2\":\" \"}}," +
       "{\"110\":{\"subfields\":[{\"b\":\"book\"}],\"ind1\":\"0\",\"ind2\":\"0\"}}]}";
 
-    testMarcUpdating(existingParsedContent, incomingParsedContent, expectedParsedContent, emptyList(), emptyList(),emptyList(),1, "100");
```

Comment:
These tags at the end of the method specify links that exist. There is only one tag in the existing record that has links - "100". So, I think that this change is not needed.

Reply:
Agree, It was needed to populate linkingRules